dev/acf-fields.json

```diff
@@ -1,5 +1,68 @@
 [
     {
+        "key": "group_58ff540088983",
+        "title": "College Fields",
+        "fields": [
+            {
+                "key": "field_58ff540c2d1ed",
+                "label": "Header Image",
+                "name": "colleges_header_image",
```

Comment:
I'd recommend just adding the Page Header Fields to colleges, instead of creating separate fields and logic for college images specifically (I think this is possible--correct me if I'm wrong).  The logic that displays header images (get_header_media_markup()) depends on other fields in the Page Header Fields group, such as the header height and -xs header image fields, to work properly on all screen sizes.

We'd probably need to split out the header title/subtitle text fields from the Page Header Fields group though to do that though.